scripts/bulk_set_requester_pays.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+if (( $# != 1 )); then
+  echo "Usage: $0 /path/to/inputfile.csv"
+  echo "inputfile.csv should contain the following column headers in order: project, bucket, toggle_type"
+  echo 'The Terra bucket paths can be formatted as "gs://fc-XXXXX" or "fc-XXXXX"'
+  echo "The toggle_type options are on|off for enabling and disabling requester pays"
+  echo "NOTE: this script requires you to be authed as your firecloud.org admin account."
+  exit 0
```

Comment:
exit 0 indicates success. we should return exit 1 for failure conditions (here and below)

Reply:
thanks! fixed that in [0fc3154](https://github.com/broadinstitute/horsefish/pull/148/commits/0fc31549ebbab88e5051a9d4ed12997192e6ccea)